serverless-test-project/package.json

```diff
@@ -19,5 +19,9 @@
     "test": "ALARM_TOPIC=test sls package",
     "test:v2": "ALARM_TOPIC=test sls package -c serverless-v2.yml"
   },
-  "gitHead": "c4300240f9e854c4eb1c5503b839882ff0cd1cae"
+  "gitHead": "c4300240f9e854c4eb1c5503b839882ff0cd1cae",
+  "dependencies": {
+    "https": "^1.0.0",
+    "sns-validator": "^0.3.4"
```

Comment:
SNS validation is probably overkill for our test, so I'd suggest to remove this.